lib/workers/pr/index.ts

```diff
@@ -45,6 +50,7 @@ export async function addAssigneesReviewers(
   if (config.assigneesFromCodeOwners) {
     assignees = await addCodeOwners(assignees, pr);
   }
+  assignees = filterUnavailableUsers(assignees);
```

Comment:
I think this would be the best place to apply the filtering because

1. the general selection logic has already be performed so we would just reduce the user set
2. it would still allow for randomized selection of sub sets afterwards
3. all assignee/reviewer logic is kept in a single place

Reply:
@rarkins (or maybe @viceice) Mind giving some early feedback here before I continue? 🙏

Reply:
Seems right